### PR TITLE
TSQL-DBCC clause

### PR DESCRIPTION
**Pull request by ohyesboy**
Reference https://msdn.microsoft.com/en-us/library/ms175008.aspx

### CODE REVIEW

**Comment by teverett** [comment]
please fix the merge conflict

**Comment by zcao** [comment]
@teverett  Resolved

**Comment by teverett** [comment]
@KvanTTT is this good to merge?

File: tsql/tsql.g4 [hunk 1]
```diff
@@ -586,6 +588,14 @@ use_statement
     : USE database=id ';'?
     ;
 
+dbcc_clause
+    : DBCC name=simple_id ('(' expression_list ')')? (WITH dbcc_options)? ';'?
```
Comment (KvanTTT) [inline tsql/tsql.g4]: Do we need for `name` label here? `simple_id` is single in this rule.

**Comment by KvanTTT** [comment]
@ohyesboy can you add yourself to contributors list in the header of file?
@teverett I added one comment but overall LGTM :)

File: tsql/tsql.g4 (hunk 1 above)
Comment (ohyesboy) [inline tsql/tsql.g4]: This is a simplified version, DBCC clause is quite versatile in MSDN https://msdn.microsoft.com/en-us/library/ms176064.aspx, there could be more simple_id added to the clause in future.

Comment (KvanTTT) [inline tsql/tsql.g4]: Ok, got it.

**Pull request merged**